libs/blocks/table/table.js

```diff
@@ -124,6 +141,8 @@ function handleAddOnContent(table) {
       el?.insertAdjacentElement(order === 'before' ? 'beforebegin' : 'afterend', tag);
     });
   });
+  window.addEventListener('resize', () => handleEqualHeight(table, '.row-heading'));
```

Comment:
We already have a `resize` event listener [here](https://github.com/adobecom/milo/blob/29a6056c98e75f4695b1e6b454e7ace7e7664df3/libs/blocks/table/table.js#L585). Could we consolidate the two?

I was also thinking whether it would be worth having this [debounced](https://github.com/adobecom/milo/blob/29a6056c98e75f4695b1e6b454e7ace7e7664df3/libs/utils/action.js#L1).

Reply:
Okay I consolidated resize events and added the debounce!